.eslintrc.json

```diff
@@ -86,6 +86,7 @@
       }
     ],
     "camelcase": "off",
+    "default-param-last": "off", // using @typescript-eslint/default-param-last instead
```

Comment:
New major version of AirBnB styleguide added this rule, so I had to turn it off because it did not support optional args with `?` (e.g. `(a = {}, a?: string) => {}`). `@typescript-eslint/default-param-last` is a bit smarter, so I added it.